packages/ui-causal-graph-editor/src/graph-viewer/causal-graph-editor.tsx

```diff
@@ -643,6 +676,11 @@ function CausalGraphEditor(props: CausalGraphEditorProps): JSX.Element {
                                 )}
                             </GraphContext.Provider>
                         </Overlay>
+                        {error && (
+                            <NotificationWrapper>
+                                <Notification notification={error} onDismiss={() => setError(null)} />
+                            </NotificationWrapper>
+                        )}
```

Comment:
It was more so that we could have a larger message to explain what happened and have it on the graph that the error happened on rather than page wide. I thought that would be easier to identify. But can change it to use onNotify if that would be better.

Reply:
How would it look like with the default notify view? I believe our storybook setup includes the global notification provider so might be worth comparing both. Otherwise this is a fine approach if it really looks better this way.

Reply:
It is just a bit shorter the default one, but the main difference is that onNotify will appear as a notification to the whole app. So if you have multiple graphs you don't know which caused the issue necessarily. Whereas this approach will always appear on the graph itself. Which do you think is better?
<img width="382" alt="Screenshot 2024-01-12 at 11 56 42" src="https://github.com/causalens/dara-ui/assets/104359539/51a5b58f-fcbd-45b7-abe7-015359e3fdc0">